src/test/lambda/explorer/standaloneNodes.test.ts

```diff
@@ -151,6 +184,62 @@ describe('DefaultStandaloneFunctionGroupNode', () => {
         }
     }
 
+    it('Sorts Lambda Function Nodes', async () => {
+        const inputFunctionNames: string[] = [
+            'zebra',
+            'Antelope',
+            'aardvark',
+            'elephant'
+        ]
+
+        ext.toolkitClientBuilder = {
+            createCloudFormationClient(regionCode: string): CloudFormationClient {
+                throw new Error('cloudformation client unused')
+            },
+
+            createLambdaClient(regionCode: string): LambdaClient {
+                return new FunctionNamesMockLambdaClient(inputFunctionNames)
+            }
+        }
+
+        const functionGroupNode = new DefaultStandaloneFunctionGroupNode(
+            new DefaultRegionNode(new RegionInfo('code', 'name'), stubPathResolver),
+            stubPathResolver
+        )
+
+        const children = await functionGroupNode.getChildren()
+
+        assert.ok(children, 'Expected to get Lambda function nodes as children')
+        assert.strictEqual(
+            inputFunctionNames.length,
+            children.length,
+            `Expected ${inputFunctionNames.length} Function children, got ${children.length}`
+        )
+
+        function assertChildNodeFunctionName(
+            actualChildNode: StandaloneFunctionNode | ErrorNode,
+            expectedNodeText: string) {
+
+            assert.strictEqual(
+                actualChildNode instanceof DefaultStandaloneFunctionNode,
```

Comment:
`instanceof` checks make me nervous, as they will fail if `actualChildNode` matches the shape of `DefaultStandaloneFunctionNode`, but wasn't created from its constructor (or the constructor of a subclass).

What are we really trying to verify here? That `node.functionName` exists. So let's test that instead.